README.md

```diff
@@ -186,6 +186,7 @@ There are some config settings you need to change in the files below.
 | `CMD_ALLOW_ANONYMOUS` | `true` or `false` | set to allow anonymous usage (default is `true`) |
 | `CMD_ALLOW_ANONYMOUS_EDITS` | `true` or `false` | if `allowAnonymous` is `true`, allow users to select `freely` permission, allowing guests to edit existing notes (default is `false`) |
 | `CMD_ALLOW_FREEURL` | `true` or `false` | set to allow new note creation by accessing a nonexistent note URL |
+| `CMD_FORBIDDEN_NODE_IDS` | `['robots.txt']` | disallow creation of notes, even if `CMD_ALLOW_FREEURL` is `true` |
```

Comment:
this is different from the config.json explanation below. Reading the code, I believe this one to be the correct description.